multiqc/modules/bcl2fastq/bcl2fastq.py

```diff
@@ -0,0 +1,124 @@
+from multiqc.modules.base_module import BaseMultiqcModule
+import logging
+import json
+from collections import OrderedDict
+from multiqc import config
+from multiqc.plots import bargraph
+
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='bcl2fastq', anchor='bcl2fastq',
+        href="https://support.illumina.com/downloads/bcl2fastq-conversion-software-v2-18.html",
+        info="bcl2fastq can be used to both demultiplex data and convert BCL files to FASTQ file formats for downstream analysis.")
```

Comment:
The `info` block always starts with the name of the module and the hyperlink. So you need to omit it at the start here as otherwise you'll end up duplicate names like this:

> [bcl2fastq](https://support.illumina.com/downloads/bcl2fastq-conversion-software-v2-18.html) bcl2fastq can be used to both demultiplex...